libs/application/templates/party-application/src/assets/CoatOfArms.tsx

```diff
@@ -0,0 +1,1152 @@
+import React from 'react'
+
+const CoatOfArms = () => (
+  <svg
```

Comment:
Can this be moved to Contentful as an asset?

Reply:
no :( i'm 97.2% positive this is the current way to display logos in the application system.. someone correct me if i'm wrong

Reply:
Currently we get our Contentful strings from the `Translation Namespace`.
I believe we only support strings atm, we could get a string that represents a url to a svg image that way.
But I feel that approach might be fragile for several reasons, so currently I think this an acceptable approach especially since we don't expect this image to change often